tensorbay/geometry/keypoint.py

```diff
@@ -13,7 +13,7 @@
 
 """
 
-from typing import Dict, List, Optional, Sequence, Type, TypeVar
+from typing import Dict, Iterable, List, Optional, Type, TypeVar
```

Comment:
Your commit message is only about `Transform3D`
The changes in `keypoint.py` is unexpected.